### DISCUSSION TITLE
I haven't received an email regarding my student pack application for more than 3 months

### DISCUSSION THREAD

**Discussion by nexosdizer** [community/community#144551] [GitHub Education]
### Hi everyone,

I had access to the GitHub Student Pack last year, but since the summer, I've lost access. When I resubmitted an application to regain access, I included a scanned certificate proving I am still a student (along with using an academic email). However, I was redirected to the home page, and in the GitHub Student Pack section, I saw the message, " **We will notify you via email when your experiences and offers become available to you.** " It's been over three months, and I still haven't received any response.

![image](https://github.com/user-attachments/assets/b1f63f2c-c546-4159-93f6-e64b96990f79)

**Comment by limitless-100** [comment]
This is really weird. I am sorry you are going through this. My advice will be to message them(The GitHub Education Team) via their email address.

**Reply by nexosdizer** [reply]
Ok thank you for you reply

**Comment by davevad93** [comment]
Hi @nexosdizer ,

Try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.

**Reply by nexosdizer** [reply]
Ok, thanks for your help. I'll try it and see if they can solve this problem.

**Reply by davevad93** [reply]
You're welcome @nexosdizer , keep in mind that it can take a while to get a reply from support, even weeks so be patient. 👍

**Reply by nexosdizer** [reply]
Hopefully, it won't take too long—I  need the domain name and the free cloud credits provided with this pack.
Thank you again for your quick responses.
